### ISSUE TITLE
Tracking how much distance I moved my cursor

### ISSUE THREAD

**Issue by Atsilut** [RoanH/KeysPerSecond#97]
It's gonna be great if it added though there is no reasonable purpose
Just my opinion, hope you think it's a good idea

**Comment by RoanH** [comment]
Yep, this will definitely be added, I'd originally planned to include panels with the total distance and current cursor speed in the last release but I ran out of time. Most likely they'll arrive with the next release, though I'm unsure when I'll get around to that.

**Comment by RoanH** [comment]
Thanks! I will need to add some sort of calibration setup to map between pixels and cm/mm. I'm not entirely certain yet how that will look like exactly, but that might be a good reference.
